Brisk/Extensions/NSStoryboard+Extension.swift

```diff
@@ -1,13 +1,13 @@
 import AppKit
 
-private let kMainStoryboard = NSStoryboard(name: "Main", bundle: nil)
+private let kMainStoryboard = NSStoryboard(name: NSStoryboard.Name(rawValue: "Main"), bundle: nil)
```

Comment:
So there’s no string one for this either?

Reply:
I couldn't find anyhing simpler…

Reply:
You can change `NSStoryboard.Name` to `.init` and that'll also solve the line length here